docs/api-testing/project-access.md

```diff
@@ -1,66 +1,85 @@
-- **Team Member**: This user belongs to a team and can create and interact with their team's projects and organization-wide projects. They cannot view projects with access restricted to an outside team. When creating a project, they will have the option to set access to be team-specific and organization-wide.
-- **Organization Admin**: This user can create and interact with all data in all projects across their organization.
+You create projects and edit project settings on the **Settings > General > Project details** page.  
 
-:::note
-You can also view access settings from your projects page.<br/><img src={useBaseUrl('img/api-fortress/2021/12/accessOrg2.png')} alt="New user UI" width="500" />
-:::
+1. In **API Testing**, on the **Projects** page, click **Create Project**. 
+1. Select an option in the **Create from** dropdown. 
+1. Optionally, select one or more **Tags**, or click **Add custom tag** to add a new tag. 
+1. Optionally, enter a **Description** of the project. 
+1. Optionally, enter **Notes** about the project. 
+1. Select the **Access** level for the project. If teams have been set up for your organization and you are an organization admin or a member of one or more teams, you can select a team from the dropdown. Otherwise, Access defaults to Organization. This selection cannot be changed once the project is saved. 
+1. Click **Save**. 
+
+### Managing Access to Projects
+
+Once a project has been created and saved, the Access (team or organization-wide access) is uneditable. If an organization admin deletes a team from Team Management before managing project access in APIT, the Access defaults to Organization, because Access is required. However, the project will still be available only to the original team and does not change to organization-wide access. Consider using one of the following options to update project access. 
+
+You have a few options if you need to change access to a project:
+
+- If access is assigned to a team, in Team Management, you can add or remove team members from the team in Team Management. This  will affect their access to projects assigned to the team. You must be an organization admin to work with teams in Team Management. 
+- If access is assigned to an organization and you need to have it limited to a team, you can consider creating a new project with the team access, and then exporting the tests from the project with organization access and importing them to the new project with team access. Then you can either delete the project with organization access, or manage the project by renaming it to indicate it is no longer being used. Make sure to [update any test schedules](/api-testing/schedule-test/) for tests in the project as well so that the tests in the archived project are not automatically run. You need to be a team member of the team you are assigning to the project or an organization admin to do this. 
```

Comment:
```suggestion
- If access is assigned to an organization and you need to have it limited to a team, you can create a new project with the team access, and then export the tests from the project with organization access and import them to the new project with team access. Then you can either delete the project with organization access, or manage the project by renaming it to indicate it is no longer being used. Make sure to [update any test schedules](/api-testing/schedule-test/) for tests in the project as well so that the tests in the archived project are not automatically run. You need to be a team member of the team you are assigning to the project or an organization admin to do this. 
```